### ISSUE TITLE
The newest libsass generated the infinity repeat classes (sometimes displayed "Killed")

### ISSUE THREAD

**Issue by YamiOdymel** [sass/libsass#1926]
```
// The amount of the column of a single row.
$columnAmount: 16

// The name of the each column size.
$columnName  : one two three four five six seven eight nine ten eleven twelve thirteen fourteen fifteen sixteen

=column-generator
  @for $i from 1 through $columnAmount
    [class*="#{nth($columnName, $i)} wide"]
      width: 100% / $columnAmount * $i 

    &[class*="#{nth($columnName, $i)} column"],
    & .row[class*="#{nth($columnName, $i)} column"]
      > .column:not(.row)
        width  : 100% / $columnAmount * $i !important

.grid
  +column-generator
```

is killed or generated a infinity class ".row.row.row.row.row" with libsass **3.3.3-24-g2acba**

the same problem happened with the latest release **3.3.3**, 

and I also **tested it with the 3.3.2, it's just fine**.

seems like it's **3.3.3** and **3.3.3-24-g2acba** cause the problem.

**Comment by mgreter** [comment]
This seems to be fixed in 3.3.4 already! //CC @xzyfer

**Comment by xzyfer** [comment]
@mgreter correct, I noticed this at the time (https://github.com/sass/libsass/pull/1925#issuecomment-187668885) but must not have updated this issue.
